packages/components/slider/src/range-slider.tsx

```diff
@@ -684,26 +693,38 @@ export const useRangeSlider = ({
       min,
       ariaValueText,
       getAriaValueText,
-      isDragging,
+      dragging,
       focusThumbOnChange,
-      isInteractive,
+      interactive,
       onBlur,
       onFocus,
       onKeyDown,
     ],
   )
 
   return {
+    dragging,
+    focused,
     getInputId,
     getMarkerId,
     getThumbId,
-    isDragging,
-    isFocused,
-    isVertical,
+    /**
+     * @deprecated Use `dragging` instead.
+     */
+    isDragging: dragging,
+    /**
+     * @deprecated Use `focused` instead.
+     */
+    isFocused: focused,
+    /**
+     * @deprecated Use `vertical` instead.
+     */
+    isVertical: vertical,
```

Comment:
@kaehehehe 
Sorry if I just don't understand.
Why did you give this section `@deprecated`?

Reply:
@koralle 
Since we are moving in a direction that does not use the prefix, shouldn't we remove that '`is`' prefix as well?

Reply:
@koralle 
Fixed. Thank you for pointing it out! : ) [ac3865c](https://github.com/yamada-ui/yamada-ui/pull/3798/commits/ac3865c2c089c4a5a803ed2387eb96d841107c0e)